packages/table/tests/react/index.js

```diff
@@ -227,6 +236,13 @@ ReactDOM.render(
 				<AUtableCell render={renderVicTags()} />
 			</AUtableRow>
 		</AUtableBody>
+		<AUtableFooter>
+			<AUtableRow>
+				<AUtableCell data="Total" type="text" />
+				<AUtableCell data="5,996,400" type="numeric" className="bold-cell" />
```

Comment:
Not sure why we're adding the `bold-cell` class here, and not on other cells in the table footer?